src/Common/PortalNotifications.ts

```diff
@@ -1,41 +1,41 @@
-import * as DataModels from "../Contracts/DataModels";

```

Comment:
>DataModels [](start = 12, length = 10)

just undo this file if no chnage?

Reply:
I am not able to. Not sure why. I copy pasted the code from master but for some reason it still shows it as modified. I checked the line endings too, they are also consistent with what is present in master

Reply:
according to what i see using codeflow, there is some spacing difference. may be the editor you are using has some settings for tabs?

Reply:
looks like some line ending issues. I checked it out again. The existing file has a Unix style line ending (which shouldn't be commitable, dont known how it got through). My change make it the standard line ending (Widnows style) seen in other files. Let me sync with steve, a rule might have been added after this got commited.

Reply:
We don't enforce line endings other than what git does. The default did change at one point. Make sure your local git instance is up to date and setup properly for line endings https://docs.github.com/en/free-pro-team@latest/github/using-git/configuring-git-to-handle-line-endings

I'm ok with checking this in. Prettier v2 has line ending checks and I can PR that upgrade after this PR is merged.